src/callstack.cr

```diff
@@ -145,14 +145,20 @@ struct CallStack
   end
 
   private def decode_backtrace
+    current_dir = Dir.current
+    current_dir += '/' unless current_dir.ends_with?('/')
+
     @callstack.compact_map do |ip|
       pc = CallStack.decode_address(ip)
 
       file, line, column = CallStack.decode_line_number(pc)
-      if file == "??"
-        file_line_column = "??"
-      else
+
+      if file && file != "??"
         next if @@skip.includes?(file)
+
+        # Turn to relative to the current dir, if possible
```

Comment:
This probably would get confusing if the executable changes it's directory mid-program. We could have multiple stacktraces from the same program have different representations for the same file. I think it'd be best to record `Dir.current` very early in the init.

Reply:
We have ˋProcess::INITIAL_PWDˋ in [in process/executable_path.cr](https://github.com/crystal-lang/crystal/blob/11e76e49f2f48f370c9e82c48c59cd8343e25838/src/process/executable_path.cr#L12) maybe we can use that? (As a constant its set very early)

Reply:
Nice find! I think we can actually expose that constant as it might be useful for others.

Reply:
I used it, but I'm not sure now we should expose it. Or maybe find another way/name and document it. So out of the scope of this PR.